src/caselawclient/Client.py

```diff
@@ -889,6 +889,13 @@ def get_version_created_datetime(self, judgment_uri: DocumentURIString) -> datet
             "%Y-%m-%dT%H:%M:%S.%f%z",
         )
 
+    def has_html_representation(self, judgment_uri: DocumentURIString) -> bool:
+        uri = self._format_uri_for_marklogic(judgment_uri)
+        vars: query_dicts.HasHtmlRepresentationDict = {
+            "uri": uri,
+        }
+        return self._eval_and_decode(vars, "has_html_representation.xqy") == "true"
```

Comment:
I don't think this file has been checked in?

Reply:
it got lost in a sea of other unstaged files I have 🤦